app/views/declaration/transport_leaving_the_border.scala.html

```diff
@@ -19,50 +19,110 @@
 @import forms.declaration.ModeOfTransportCode._
 @import forms.declaration.TransportLeavingTheBorder
 @import models.requests.JourneyRequest
-@import views.Title
-@import views.components.inputs.RadioOption
+@import uk.gov.hmrc.govukfrontend.views.html.components.{Empty => _, _}
```

Comment:
Why this rename `Empty => _` or I understand it incorrectly?

Reply:
Because there is an `Empty` class in ModeOfTransportCode that conflicts with the one in `govukfrontend`